src/semeio/workflows/ahm_analysis/ahmanalysis.py

```diff
@@ -98,27 +105,49 @@
 """
 
 
-class AhmAnalysisJob(SemeioScript):
+class AhmAnalysisJob(ErtScript):
     """Define ERT workflow to evaluate change of parameters for eac
     observation during history matching
     """
 
     def run(
         self,
-        target_name="analysis_case",
-        prior_name=None,
-        group_by="data_key",
-        output_dir=None,
-    ):
-        # (SemeioScript wraps this run method)
-
+        storage: Storage,
+        config_file: str,
```

Comment:
Related to comment below, we might consider just passing an output path to workflows instead. Think that would be cleaner?